README.md

```diff
@@ -1,110 +1,24 @@
-<p align="right">
-    <a href="https://badge.fury.io/rb/just-the-docs"><img src="https://badge.fury.io/rb/just-the-docs.svg" alt="Gem version"></a> <a href="https://github.com/just-the-docs/just-the-docs/actions/workflows/ci.yml"><img src="https://github.com/just-the-docs/just-the-docs/actions/workflows/ci.yml/badge.svg" alt="CI Build status"></a> <a href="https://app.netlify.com/sites/just-the-docs/deploys"><img src="https://api.netlify.com/api/v1/badges/9dc0386d-c2a4-4077-ad83-f02c33a6c0ca/deploy-status" alt="Netlify Status"></a>
-</p>
-<br><br>
-<p align="center">
-    <h1 align="center">Just the Docs</h1>
-    <p align="center">A modern, highly customizable, and responsive Jekyll theme for documentation with built-in search.<br>Easily hosted on GitHub Pages with few dependencies.</p>
-    <p align="center"><strong><a href="https://just-the-docs.github.io/just-the-docs/">See it in action!</a></strong></p>
-    <br><br><br>
-</p>
+# Just the HM Docs
 
-<p align="center">A video walkthrough of various Just the Docs features</p>
+Just the HM Docs is a GitHub Pages template developed for the purpose of quickly deploying documentation for Human Made open source projects. It is a fork of the popular [Just the Docs](https://github.com/just-the-docs/just-the-docs) Jekyll theme, which provides a robust and thoroughly-tested foundation for your website. Just the HM Docs includes features such as:
 
-https://user-images.githubusercontent.com/85418632/211225192-7e5d1116-2f4f-4305-bb9b-437fe47df071.mp4
+- automatic [navigation structure](https://humanmade.github.io/just-the-hm-docs/docs/navigation-structure)
+- instant, full-text [search](https://humanmade.github.io/just-the-hm-docs/docs/search) and page indexing
+- and a set of [UI components](https://humanmade.github.io/just-the-hm-docs/docs/ui-components) and authoring [utilities](https://humanmade.github.io/just-the-hm-docs/docs/utilities)
 
-## Installation
+## Getting Started
 
-### Use the template
+Getting started with Just the HM Docs is simple.
 
-The [Just the Docs Template] provides the simplest, quickest, and easiest way to create a new website that uses the Just the Docs theme. To get started with creating a site, just click "[use the template]"!
+1. Create a [new repository based on Just the HM Docs](https://github.com/humanmade/just-the-hm-docs/generate).
```

Comment:
Feels like this reflects the classroom theme repo process -- didn't we decide to move towards using this more as a [remote_theme](https://docs.github.com/en/pages/setting-up-a-github-pages-site-with-jekyll/adding-a-theme-to-your-github-pages-site-using-jekyll), so that you could add a config pointing to this repo to a given project's `docs/_config.yml` and GH would pick it up there? (assuming `docs/` is the publishing source, could also of course be GH pages).

may not be necessary to change at this time, but want to revisit it once we figure out the lowest-friction way to deploy this

Reply:
Yeah, I want to re-write the getting started docs, but I'm not clear enough on what the steps are yet.